tests/v2/validation/provisioning/rke1/README.md

```diff
@@ -45,10 +45,10 @@ provisioningInput:
       worker: true
       drainBeforeDelete: true
       quantity: 2
-  rke1KubernetesVersion: ["v1.27.10-rancher1-1"]
+  rke1KubernetesVersion: ["v1.28.10-rancher1-1"]
   cni: ["calico"]
   providers: ["linode", "aws", "do", "harvester", "vsphere", "azure"]
-  cloudProvider: "external-aws"
+  cloudProvider: "" # either: external-aws|rancher-vsphere
```

Comment:
Can this be expanded upon more? Something like `Currently provides support for AWS and vSphere. The supported values are `external-aws | rancher-vsphere`.

Reply:
we don't usually provide that much context in the comments of the readme. i.e. `providers` and `psact` don't have reasoning for what's listed

Reply:
also there's a whole section later on for what is supported, so I don't feel that adding further comments here is helpful. wdyt?

Reply:
I don't feel strongly of it. I was more looking at perspective if someone doesn't read the other section. If you think it's fine to leave as is, that's fine with me.